configure.cmd

```diff
@@ -50,14 +50,11 @@ if defined ROS_ARCH (
     cl 2>&1 | find "x86" > NUL && set ARCH=i386
     cl 2>&1 | find "x64" > NUL && set ARCH=amd64
     cl 2>&1 | find "ARM" > NUL && set ARCH=arm
-    cl 2>&1 | find "16.00." > NUL && set VS_VERSION=10
-    cl 2>&1 | find "17.00." > NUL && set VS_VERSION=11
-    cl 2>&1 | find "18.00." > NUL && set VS_VERSION=12
```

Comment:
I request to undo this check. Maybe you can generate a warning instead a hard failure.
The warning could say that VS2015 is our main compiler now and that automated builds do not cover VS<2015 anymore, but that PRs with fixes for maintaining/restoring older compiler support down to VS2010 would be welcome.

Reply:
No, they won't be welcome. They would actively counter our efforts to remove code duplication/hacks for VS2010 (already happened) and bring in a modern C++ library (as Dominik2 is doing right now), which will then lead to a lot more readable code and resolving the even bigger mess that our C++ build environment is right now.
No C++ developer of today wants to be bound by the limits of ancient C++03, and as soon as the first C++11 code is in, there is simply no way for VS2010 support without maintaining two versions of the same application.

If you want ReactOS to become a thing in a lifetime, you at least have to give developers the best tools available!